src/components/VContentLink/VContentLink.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <button
+    class="bg-white border border-dark-charcoal/20 rounded-sm flex hover:bg-dark-charcoal hover:text-white focus:bg-white focus:ring focus:ring-pink focus:outline-none focus:shadow-ring focus:text-black overflow-hidden"
+    :class="[
+      isStacked
+        ? 'flex-col items-start py-4 ps-4 pe-12'
+        : 'flex-row justify-between items-center w-full min-w-[22rem] p-6',
+    ]"
+    role="radio"
+    type="button"
+    :aria-checked="isSelected"
+    @click="$emit('selected', mediaType)"
+  >
+    <div
+      class="flex"
+      :class="[isStacked ? 'flex-col items-start' : ' flex-row items-center']"
+    >
+      <VIcon :icon-path="iconPath" />
+      <p class="font-semibold" :class="[isStacked ? 'pt-1' : 'ps-2 text-2xl']">
+        {{
+          isStacked
+            ? $t(`search-tab.${mediaType}`)
+            : $t(`search-tab.see-${mediaType}`)
+        }}
+      </p>
+    </div>
+    <span :class="{ 'text-sr': !isStacked }">{{ resultsCountLabel }}</span>
+  </button>
+</template>
+
+<script>
+import { computed, defineComponent } from '@nuxtjs/composition-api'
+import { resultsCount } from '~/composables/use-i18n-utilities'
+import { AUDIO, IMAGE, mediaTypes } from '~/constants/media'
+import VIcon from '~/components/VIcon/VIcon.vue'
+
+import audioIcon from '~/assets/icons/audio-wave.svg'
+import imageIcon from '~/assets/icons/image.svg'
+
+export default defineComponent({
+  name: 'VContentLink',
+  components: { VIcon },
+  props: {
+    /**
+     * One of the media types supported.
+     */
+    mediaType: {
+      type: String,
+      required: true,
+      validator: (val) => mediaTypes.includes(val),
+    },
+    /**
+     * The number of results that the search returned.
+     */
+    resultsCount: {
+      type: Number,
+      required: true,
+    },
+    /**
+     * Whether the indicated media type is currently selected.
+     */
+    isSelected: {
+      type: Boolean,
+      default: false,
+    },
+    /**
+     * `stacked` intended for mobile and `horizontal` for desktop.
+     */
+    layout: {
+      type: String,
+      default: 'stacked',
+      validator: (val) => ['stacked', 'horizontal'].includes(val),
+    },
```

Comment:
It's a shame we have changes other than just organization that get tied to this (like font size I think?) because it seems like something that `flex-wrap` should be able to take care of.

Do we envision the usage of this prop to be a situation where the component is rendered in distinct places where the user will know whether it's a mobile or desktop context? Or are we potentially just pushing the media-query up the stack so that the code for this prop will be something like like `:layout="isMinScreen('md') ? 'horizontal' : 'stacked'"`?

If it's the last case then my inclination would be to just move that logic into the component and hide the fact that it has different layouts depending on screen size from the user (so the user doesn't have to worry about it).

This is not a blocker or even a solid suggestion, just wondering what the intention is.

Reply:
I also started thinking we might be able to handle this with just css-based media queries. I can look at that when I implement this component in #530 and my 'all' results PR.

Reply:
Using the `isMinScreen()` is giving me mismatched HTML problems when the component is server-side rendered. I'll have to take a closer look at how those errors are being handled but for now I'd like to tie this together. I agree that it would be ideal to hide this in the component logic and save us a property here as well.